Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM resin/armv7hf-debian:jessie
+
+RUN apt-get update \
+	&& apt-get install -y axel wget htop nano libjpeg-dev libconfig++-dev\
+	libevent-dev git freebsd-glue curl python bison flex\
+	&& apt-get clean \
+        && apt-get autoremove -qqy 
+
+RUN curl -sL https://deb.nodesource.com/setup | bash -
+RUN apt-get install -y build-essential nodejs
+
+RUN mkdir -p /opt/adapteva && cd /opt/adapteva \
+	&& axel -n 10 http://ftp.parallella.org/esdk/beta/esdk.2014.11.20150522_linux_armv7l.tar.gz && tar -xf esdk.2014.11.20150522_linux_armv7l.tar.gz && rm esdk.2014.11.20150522_linux_armv7l.tar.gz \
+	&& ln -sTf /opt/adapteva/esdk.2014.11 /opt/adapteva/esdk
+
+ENV EPIPHANY_HOME /opt/adapteva/esdk
+
+# Enable default setup from webterminal
+RUN sed -i 's/\/bin\/sh/\/bin\/bash/g' /opt/adapteva/esdk/setup.sh && echo "source /opt/adapteva/esdk/setup.sh" >> ~/.bashrc
+
+RUN mkdir -p /app
+
+# Build libelf 0.8.13
+RUN cd /app && wget http://www.mr511.de/software/libelf-0.8.13.tar.gz \
+	&& tar -xf libelf-0.8.13.tar.gz \
+	&& cd libelf-0.8.13 && ./configure && make && make install
+
+# Build COPRTHR
+RUN bash -c "source /opt/adapteva/esdk/setup.sh && cd /app \
```

Comment:
Same drill. Also you don't need to wrap everything in a string and run via `/bin/bash`

```Dockerfile
ENV COPRTHR_VERSION parallellocalypse

RUN . /opt/adapteva/esdk/setup.sh \
    && mkdir -p /usr/src/libcoprthr \
    && curl -sL https://github.com/olajep/coprthr/archive/$COPRTHR_VERSION.tar.gz | tar xz -C /usr/src/libcoprthr --strip-components=1 \
    && cd /usr/src/libcoprthr \
    && ./configure --enable-epiphany \
    && make \
    && make install
```
I guess you could even delete `/usr/src/libcoprthr` after installation?